setup.py

```diff
@@ -82,6 +83,7 @@
             'pytest-cov',
             'pytest-django',
             'pytest-factoryboy',
+            'pytest-memray',
```

Comment:
If we're adding this, we could also consider adding `--memray` to the pytest invocation in `tox.ini`. WDYT @AlmightyYakob? If not, I think it's at least worth mentioning in the README that you can do `tox -e test -- --memray` since this plugin is installed.

Reply:
I considered that, I think the latter approach (documenting the option in the README) is the right approach. I'll add that.